components/toast/toast.css

```diff
@@ -0,0 +1,36 @@
+.toast {
+  position: fixed;
+  left: 1rem;
+  bottom: 1rem;
+  background-color: var(--white);
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+  border-radius: 5px;
+  box-shadow: rgb(0 0 0 / 12%) 0 0 5px;
```

Comment:
hey you can use variable for box shadows too
btw what is this rgb(0 0 0 / 12%) I am curious 😅 
alpha value?